lib/cash-in/cash-in-tx.js

```diff
@@ -30,10 +31,11 @@ function post (machineTx, pi) {
 
           if (_.some(it => it.created_by_operator === true)(blacklistItems)) {
             blacklisted = true
+            notifier.blacklistNotify(r.tx, false)
           } else if (_.some(it => it.created_by_operator === false)(blacklistItems) && rejectAddressReuseActive) {
```

Comment:
I know this is existing code, but can't we simplify the check using (it => it.created_by_operator) and (it => !created_by_operator)?

```suggestion
          if (_.some(it => it.created_by_operator)(blacklistItems)) {
            blacklisted = true
            notifier.blacklistNotify(r.tx, false)
          } else if (_.some(it => !it.created_by_operator)(blacklistItems) && rejectAddressReuseActive) {
```